utils.js

```diff
@@ -208,7 +209,7 @@ class Exemplar {
 
   // TODO both these methods should copy
   getImages() {
-    return this.images;
+    return param.randomTriplets ? this.images.sort( (x, y) => Math.random() - 0.5): this.images;
```

Comment:
Also, are you sure this will give each image a 33% chance of appearing anywhere? It's late so I don't want to look into this a lot right now but it seems like there might be some room for weirdness here, depending on how the JS sort function works.